docs/lambda-k-pi/index.md

```diff
@@ -0,0 +1,14 @@
+# pγ → ΛK⁺π⁰
+
+We are doing amplitude analysis and Partial Wave Analysis (PWA) with [CAS-assisted model building](https://compwa.github.io/symbolics), which is a techniqued employed in the ComPWA project.
+
+- The first demonstration example is:<br>
+  $p \gamma \to$ Resonances (e.g. N\*) $\to \eta \pi^0 p$
+- The main focus of this research would be:<br>
+  $p \gamma \to$ Resonances (e.g. N\*, K\*) $\to K^+ \pi^0 \Lambda$
+
+Current long-term plan: This research project focuses on developing and applying amplitude models to analyze data from the GlueX experiment, specifically targeting $N^\*$ and $K^\*$ resonances.
```

Comment:
These `*` look weird. But anyway, I would remove the text altogether, see https://github.com/ComPWA/gluex-nstar/pull/77/files#r1715321666

![image](https://github.com/user-attachments/assets/e04061d7-1f51-4056-b17e-0f8ba964dacf)